### PR TITLE
add raw transaction

### PR DESCRIPTION
**Pull request by ldrygala**

### CODE REVIEW

File: src/main/scala/com/bitcoinpaygate/bitcoin4s/BitcoinClient.scala [hunk 1]
```diff
@@ -65,7 +66,7 @@ class BitcoinClient(httpClient: HttpClient)(implicit system: ActorSystem, materi
   }
 
   def listUnspentTransactions(minimumConfirmations: Option[Int] = None, maximumConfirmations: Option[Int] = None)(implicit executionContext: ExecutionContext): Future[BitcoinResponse[UnspentTransactions]] = {
-    val request = httpClient.httpRequestWithParams("listunspent", Vector(minimumConfirmations.getOrElse(0), maximumConfirmations.getOrElse(0)))
+    val request = httpClient.httpRequestWithParams("listunspent", Vector(minimumConfirmations.getOrElse(1), maximumConfirmations.getOrElse(9999999)))
```
Comment (lukeindykiewicz) [inline src/main/scala/com/bitcoinpaygate/bitcoin4s/BitcoinClient.scala]: Do you remember @wlk if there was a reason in having zeros here?

File: src/test/resources/create-raw-transaction.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "result": "02000000000180969800000000001976a914f5b32cc7579d678b60780846128b0f98f74cd10e88ac00000000",
+  "error": null,
+  "id": null
+}
```
Comment (lukeindykiewicz) [inline src/test/resources/create-raw-transaction.json]: Why idea doesn't add newline ? Isn't that some kind of "standard"?

Comment (lukeindykiewicz) [reply src/test/resources/create-raw-transaction.json]: Just treat it as my periodic reminder why to not use IDE ;p

File: src/main/scala/com/bitcoinpaygate/bitcoin4s/BitcoinClient.scala (hunk 1 above)
Comment (wlk) [reply src/main/scala/com/bitcoinpaygate/bitcoin4s/BitcoinClient.scala]: https://bitcoin.org/en/developer-reference#listunspent
0 is required to include transactions with 0 confirmations, 9999999 is the default.

IMO you shouldn't change this

Comment (ldrygala) [reply src/main/scala/com/bitcoinpaygate/bitcoin4s/BitcoinClient.scala]: i was thinking that when someone didn't set min and max we should use default not `0` for both

Comment (ldrygala) [reply src/main/scala/com/bitcoinpaygate/bitcoin4s/BitcoinClient.scala]: i totally agree with @lukeindykiewicz, i was thinking that we need to set something therefor i chose bitcoind defaults

**Pull request merged**